sklearn/preprocessing/tests/test_data.py

```diff
@@ -100,6 +100,27 @@ def test_scaler_1d():
     X = np.ones(5)
     assert_array_equal(scale(X, with_mean=False), X)
 
+    X = np.zeros(8) + np.log(1e-5)
```

Comment:
I don't understand this number. Isn't this a fancy way of saying -11. That is not what you intended, right?